Bio/Nexus/Trees.py

```diff
@@ -594,8 +594,7 @@ def display(self):
         table = [
             ("#", "taxon", "prev", "succ", "brlen", "blen (sum)", "support", "comment")
         ]
-        # Sort this to be consistent across CPython, Jython, etc
-        for i in sorted(self.all_ids()):
```

Comment:
I think this should still be explicitly sorted - different versions of CPython and PyPy are not consistent.

Reply:
Are you sure?

Reply:
We may be OK on CPython 3.6+ (we don't support older Pythons anymore, which had different dict sorting) and recent PyPy.